PatientRegistrationApp/src/App.vue

```diff
@@ -1,616 +1,830 @@
 <template>
-  <v-app id="Patient-registration">  
+  <v-app id="Patient-registration">
```

Comment:
Do you have an auto formatter installed which is why you're showing so much changes when I don't think there are actually a lot?